axolotl-web/src/components/VueOtpBox.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <div id="wrapper" :style="{ justifyContent: direction }">
+    <div v-for="(char, index) in arraySize" v-bind:key="'otp_' + index">
+      <input
+        class="inputBox"
+        type="tel"
+        :key="index"
+        maxlength="1"
+        v-model="arraySize[index]"
+        @keydown="
+          handleEnterKey($event);
+          handleKeyDown($event, index);
+        "
+        @input="handleInput($event, index)"
+        @paste="onPaste"
+        :autofocus="index === 0"
+        :ref="`otp${index}`"
+        @focus="newColor(index)"
+        @blur="defaultColor(index)"
+      />
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: "VueOtp",
+  props: ["numberOfBoxes", "position", "color", "clearInput"],
+
+  data() {
+    return {
+      arraySize: undefined,
```

Comment:
I notice that undefined is used on a few places in the codebase. Instead of using undefined, I would however argue that using null is best practice, and that if possible we should aim to use null over undefined.

> Null: It is the intentional absence of the value. It is one of the primitive values of JavaScript.
> Undefined: It means the value does not exist in the compiler. It is the global object.

https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/

Reply:
`Null` is better than undefined, because defined undefined is weird ;)